### DISCUSSION TITLE
Hope to edit environment name Vercel deploy on GitHub integration

### DISCUSSION THREAD

**Discussion by moh3n9595** [vercel/community#1476] [Idea]
Hi, I have a GitHub repo and want to deploy multiple projects on it. But I'm not too fond of the default environment naming Vercel use. `(Production - PROJECT_NAME_IN_VERCEL_DASHBOARD)`. Can we customize this naming convention?

**Comment by amyegan** [comment]
Hi, Mohsen! Are you talking about the bot comment labels showing what was deployed?
![image](https://user-images.githubusercontent.com/8247105/217635661-00f3e120-fe4f-4b92-84e6-d5847f8d6f48.png)

Or is it something else that you want to customize?

**Reply by moh3n9595** [reply]
Yes, I resolved my issue by triggering the deployment process with GitHub actions but it can be a good feature to handle this with the Vercel bot.

**Reply by mia-iofinnet** [reply]
Hi, @moh3n9595! Would you mind sharing what you did in GitHub Actions to achieve this? 🤔
